src/Search/Search.js

```diff
@@ -2,133 +2,35 @@ import React, { useState, useEffect } from 'react';
 import { useNavigate } from 'react-router-dom';
 import Card from '../Card/Card';
 import './Search.css'
+import useSearchResults from '../hooks/useSearchResults';
 
 function WikipediaSearch() {
   const [searchInput, setSearchInput] = useState('');
-  const [initialResults, setInitialResults] = useState({});
-  const [page, setPage] = useState([]);
-  const [controversies, setControversies] = useState([]);
+  const { initialResults, controversies, triggerSearch } = useSearchResults();
   const navigate = useNavigate();
 
-  const handleInputChange = (e) => {
-    setSearchInput(e.target.value);
-  };
-
   const handleFormSubmit = (e) => {
     e.preventDefault();
     if (searchInput.trim() !== '') {
-      displaySearchResults(searchInput);
+      triggerSearch(searchInput);
     }
     setSearchInput('');
   };
   
-  async function displaySearchResults(searchTerm) {
-    const url = 'https://en.wikipedia.org/w/api.php?';
-    let params = {
-      action: 'query',
-      list: 'search',
-      format: 'json',
-      origin: '*',
-      srlimit: 20,
-      srsearch: searchTerm,
```

Comment:
its awesome you were able to move these to a different space and then still be able to call on them, that makes things look super streamlined